Source/Particles/Pusher/PushSelector.H

```diff
@@ -40,6 +40,8 @@
  * \param t_chi_max                 Cutoff chi for QSR
  * \param dt                        Time step size
  */
+
+template <int do_sync>
```

Comment:
Please add Doxygen for `\tparam do_sync`, since it should be clear from the API if we mean Synchrotron QED (or some sync in compute).

Reply:
done